crates/typst/src/symbols/sym.rs

```diff
@@ -842,6 +842,7 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
      bet: 'ב',
      gimmel: 'ג',
      gimel: 'ג',
+     dalet: 'ד',
```

Comment:
Since we have two variants for all the other ones, maybe we should also have `daleth`, as it is called in Unicode? Does that make sense?